exhibit/models.py

```diff
@@ -1,11 +1,13 @@
 from django.db import models
 from wagtail.core.models import Page
 from wagtail.core.fields import RichTextField
+from wagtail.core.templatetags import wagtailcore_tags
```

Comment:
is this needed here if it's getting imported in `serializers.py?`

Reply:
Nope! Removed.